scripts/python/packages/parthenon_tools/setup.py

```diff
@@ -22,8 +22,8 @@
     description="Python helper scripts to be used with Parthenon",
     long_description=long_description,
     url="https://github.com/lanl/parthenon/tree/develop/scripts/python/parthenon_tools",
-    author="Josh Dolence, Philipp Grete, Andrew Gaspar, Joshua S. Brown",
-    author_email="joshbro42867@yahoo.com",
+    author="The Parthenon Team",
+    author_email="",
```

Comment:
Josh Brown has moved on so his email should probably not be here. That said, should we have an official Parthenon email?